Sources/Kitchen.swift

```diff
@@ -80,6 +82,15 @@ extension Kitchen {
         }
     }
 
+    public static func serveInNewWindow(urlString urlString: String, type: PresentationType = .Default) {
+        temporaryWindow = UIWindow(frame: UIScreen.mainScreen().bounds)
+        Kitchen.navigationController.setViewControllers([UIViewController()], animated: true)
+        Kitchen.navigationController.delegate = sharedKitchen
```

Comment:
Too bad we got an error...
```
Changing TVNavigationController's UINavigationControllerDelegate is not recommended
 because it is itself the delegate in order to implement its UIViewController (TVNavigationAdditions)
```